### PR TITLE
Added permalink to Recent Contributors section

### LINKED ISSUE #460: Add permalink to Recent Contributors section
Hi, this is a [first-timers-only issue](https://publiclab.github.io/community-toolbox/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** the file [index.html](https://github.com/publiclab/community-toolbox/blob/main/index.html) in the `community-toolbox` repository (press the little pen Icon) and edit the line as shown below. 

[See this page](https://publiclab.github.io/community-toolbox/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
@@ -78,7 +78,8 @@
                 <ul>
                     <li><a class="line" href="#candidate">Candidates</a></li>
                     <li><a class="line" href="#fto-author">FTO Authors</a></li>
-                    <li><a class="line" href="#contributer">Contributers</a></li>
+                    <li><a class="line" href="#recent-contributer">Recent contributors</a></li>
+                    <li><a class="line" href="#contributer">Contributors</a></li>
                     <li><a class="line" href="#lang-sec">Languages Used</a></li>
                 </ul>
             </li>
@@ -247,7 +248,7 @@ <h2 id="stale-head">Stale Issues</h2>
       </div>
 
 
-      <h2 class="recent-contributors-head">Recent Contributors</h2>
+      <h2 id="recent-contributors" class="recent-contributors-head">Recent Contributors</h2>
 
       <div class="recent-contributors">
         <div style="display: flex; justify-content: space-between">
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### 🤔❓ Questions?

Leave a comment below!

### Is someone else already working on this?

We encourage you to link to this issue by mentioning the issue # in your pull request, so we can see if someone's already started on it. **If someone seem stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://publiclab.github.io/community-toolbox/#r=all). Thanks!

(This issue was created by [First-Timers-Bot](https://github.com/hoodiehq/first-timers-bot).)

### PR DESCRIPTION
**Pull request by swetshaw**
Updated recent contributors list and heading
fixes #460 - Add permalink to Recent Contributors section

### CODE REVIEW

**Comment by daemon1024** [comment]
Hi @swetshaw, Thanks for working on this. 

As I see, there's been already been approval at https://github.com/swetshaw/community-toolbox/pull/1 🎉 . It would be great if you could update the Pull Request Title here as well to something more descriptive.

Thanks Again 🥳

**Comment by swetshaw** [comment]
Hi @daemon1024 , I have updated the Pull Request Title here. Please check if it looks good. Thankyou. :)

File: index.html [hunk 1]
```diff
@@ -78,7 +78,8 @@
                 <ul>
                     <li><a class="line" href="#candidate">Candidates</a></li>
                     <li><a class="line" href="#fto-author">FTO Authors</a></li>
-                    <li><a class="line" href="#contributer">Contributers</a></li>
+                    <li><a class="line" href="#recent-contributer">Recent contributors</a></li>
++                    <li><a class="line" href="#contributer">Contributors</a></li>
```
Comment (TildaDares) [inline index.html:82]: Hi @swetshaw, can you remove the extra plus sign you added to line 82? Thanks!

**Review by TildaDares — CHANGES_REQUESTED** [review]

**Comment by TildaDares** [comment]
Thank you @swetshaw!!

**Pull request merged**